Blockchain/readme.md

```diff
@@ -95,3 +98,4 @@
   - [Alchemy (Youtube)](https://www.youtube.com/c/AlchemyPlatform) 
   - [Dapp University(Youtube)](https://www.youtube.com/channel/UCY0xL8V6NzzFcwzHCgB8orQ)
   - [Patrick Collins(Youtube)](https://www.youtube.com/c/patrickcollins)
+  - [EatTheBlocks(Youtube)](https://www.youtube.com/@EatTheBlocks)
```

Comment:
Would be better if you could point to a specific video instead of a channel.

Reply:
It's a good channel with structured playlists. I think adding the channel is more preferable than a singular video . Folks can explore the channel and learn accordingly. If you still want single video than do let me know. We can add
10 hr Full Ethereum Video by Dapp University
32 hr Full Stack Web3 Development Video by Patrick in freeCodeCamp and many more

Reply:
I think it would be better to add both the video as well as the channel

Reply:
Please check added both the channel and the specific video for that channel(Defi Tutorial which is not easily available on YT).